src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -101,6 +101,11 @@ def _init_weights(self, module):
 
             - 1 indicates the head is **not masked**,
             - 0 indicates the head is **masked**.
+        image_patches (`torch.FloatTensor` of shape `(batch_size, num_total_patches, patch_size_ x patch_size x num_channels )`, *optional*):
+            Image patches to be used as continuous embeddings. The patches are flattened and then projected to the
+            hidden size of the model.
+        image_patches_indices (`torch.LongTensor` of shape `(batch_size, num_total_patches, patch_size_ x patch_size x num_channels )`, *optional*):
+            To do
```

Comment:
to do

Reply:
The shape is not guaranteed to be `(batch_size, num_total_patches, patch_size_ x patch_size x num_channels )`. It would be something like `(batch_size, num_total_patches + number_of_newline_tokens + number_of_text_tokens, patch_size_ x patch_size x num_channels )`, where the number of newline tokens depends on the image size. 

Then 

"Indices indicating at which position the `image_patches` have to be inserted in `input_embeds`."